### ISSUE TITLE
The 3d model floats like half a metter above ground when grounded

### ISSUE THREAD

**Issue by IAHM-COL** [IAHM-COL/727-230#12]
Launching the JSB advanced version I noticed the gears are not really in contact with ground :?

I had not evaluated whehter this is also a problem in the yasim.

IH-COL

**Comment by JWocky** [comment]
No problem in Yasim, in the JSBSIM version, I have still to program the mass before I can balance her on the ground and calibrate the dampers. Just something, that isn't done yet.

**Comment by IAHM-COL** [comment]
ok
Good :) 

I just noticed this, and I was not sure. Take your time :+1:

**Comment by JWocky** [comment]
Problem: Sanhozay did the instruments-file and I tried to dock onto that and in the process, I found, there is no mk-rmi-adf-nav.nas as he defined in the Nasal section. There isn't even one in the 707 he used as example. So is there something in work, I missed, or is this a mistake?

**Comment by IAHM-COL** [comment]
Hi JWocky

The info about when those nas where removed due to legacy is in the information of this commit
https://github.com/IAHM-COL/727-230/commit/4ffb144d1886376f10d8ecb113bc00b1d905d969

If you need the files back let me know an I restore them for you
I understand the files were replaced by other alternative by Sanhozay

**Comment by IAHM-COL** [comment]
So, If I read correctly, that nasal file is no longer needed, to manage the ADF and RMI needles. 
The solution seem to not call that file anymore. 

Sanhozay may be able to provide deeper insight.

**Comment by JWocky** [comment]
Okay, I just dock on to Sanhozay's alternatives.

**Comment by IAHM-COL** [comment]
Sounds good. 
If the file is outdated, is better out that in.

I am watching this going on as it happens. Why?
Because you two are working on similar set of files, and I predict a "merge" conflict. 

Merge conflicts are a very important part of the git flow, so I ask both @sanhozay and @JWocky  not to panic if the git tells "merge conflic" and let me know as soon as it comes thru. 

For now keep in mind:
While working with colaborators, always pull _succesfully_ before attempting to push

**Comment by JWocky** [comment]
Not really, I do my stuff, pull and merge here, sort it out manually and then push it back

**Comment by sanhozay** [comment]
The confusion over the -set.xml files is my fault because I didn't realize there were two different ones when I made changes. When you are ready, we can reorganize the -set.xml files. If it's causing you a problem, let me know and I'll reconcile them, otherwise I'll leave things alone.

Don't worry about that  mk-rmi-adf-nav.nas file. It's not required for the 727. It was copied from the Connie and it's not required there either. It works around bugs that don't exist so was just wasting CPU.

**Comment by JWocky** [comment]
I have already docked on to your instruments-file in init and will try to dock on to your controls-file (I have currently controls and controls[1], so it's a bit messy). But I have the JSB-set in work at the moment and hope, till Friday, I can get ore life into that autopilot of mine. So, I am the one who has to pull, merge and get it together again, since I sit three days on the files without push ;-)
